classes/task/reminder.php

```diff
@@ -52,16 +52,17 @@ public function get_name() {
     public function execute() {
         global $DB;
         $kanbancards = $DB->get_records_sql(
-            'SELECT ' . $DB->sql_concat('c.id', '"-"', 'a.userid') . ' as uniqid,
+            'SELECT ' . $DB->sql_concat('c.id', '\'-\'', 'a.userid') . ' as uniqid,
                     c.id as id, c.title as title, k.name as boardname, c.duedate as duedate, a.userid as userid, k.id as instance
                FROM {kanban_card} c
          INNER JOIN {kanban_assignee} a ON a.kanban_card = c.id
                 AND c.duedate != 0
                 AND c.reminder_sent = 0
                 AND c.completed = 0
-                AND (c.duedate < now() OR (c.reminderdate != 0 AND c.reminderdate < now()))
+                AND (c.duedate < :time OR (c.reminderdate != 0 AND c.reminderdate < :time2))
          INNER JOIN {kanban_board} b ON b.id = c.kanban_board
-         INNER JOIN {kanban} k ON b.kanban_instance = k.id'
+         INNER JOIN {kanban} k ON b.kanban_instance = k.id',
+            ['time' => time(), 'time2' => time()]
```

Comment:
I suggest using a variable `$time = time();` and inject its value into the statement. If these two time calls differ very slightly and this statement ever shows up in an analysis tool one might wonder what these different times exactly mean :-)